std_msgs/Header header
widowx_block_manipulation/BlockPose[] poses